README.md

```diff
@@ -1,2 +1,77 @@
+
+The Agile movement seeks to make a change to outdated project management by introducing a new method in which to control and structure projects, whether they are in the games industry or in schools. The way the Agile movement does this is by introducing a concept called scrum, which is to be introduced when managing projects. The basis of scrum is to promote critical feedback, team self-management and to make playable or working prototypes within a short amount of time [1]. 
+
+Within the games industry the use of scrum is very prominent because it is seen as risk management by many companies. This is because of how involved the teams need to be to initiate a Scrum, they need to be updating the Scrum master almost every day with information on what they have done yesterday, what they are doing today and what is hindering them from doing work. This promotes flexibility to a project because of how easy it is to respond to a change within the project [2]. Therefore, Scrum should be used to further education in schools and bodies of higher education. In the study by Kropp, Meier and Biddle [3], applied agile to a real classroom environment, the most difficult parts of the agile methodology to students were customer interaction and skills and culture. The way this can be counteracted within the school environment is that the teacher be the customer and provide specifications to a project this would allow the students to sculpt the project that they are doing with the teacher, perfectly because the teacher would be the ideal customer. The other problem seen with the Agile movement in classrooms is that students have less experience in agile product management than a professional. This can be overcome though by introducing Scrum in schools from a young age and having it done by the whole school as a new method of teaching. This means that students can gain the experience they need passively, instead of the methodology being a whole new concept to them. The problem with this however is that the school curriculum would need to be changed to better match the Scrum ideology. A point brought up in the study by Kropp, Meier and Biddle [3] is that students are not co-located, this problem is easily overcome through the use the internet and programs such as Trello which allows students to post their work onto it and check who is doing what during a collaborative project and Skype which let students conduct stand-up meeting even though they are not in the same location. 
```

Comment:
You could link education to gamification rather than meandering off into the games industry. I recommend that you flick through [The Multiplayer Classroom by Lee Sheldon](https://books.google.co.uk/books?hl=en&lr=&id=qYMLAAAAQBAJ&oi=fnd&pg=PP1&dq=multiplayer+classroom&ots=Do_j2e_Kek&sig=o3he5Pf3g7w1apXt6RGWUxPac6A#v=onepage&q=multiplayer%20classroom&f=false) for ideas.